kube/objects.go

```diff
@@ -24,6 +24,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Comment:
Can you please add a comment above this import statement to show indicate why it is unused?

Reply:
```go
	// Load client-go authentication plugins
	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Does this clarify the reason enough? I thought about adding the link as a reference, but I'm not sure.